Building DAG of jobs...
Provided cores: 32
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	6	AnalyzeCovariates_MT
	6	AnalyzeCovariates_genome
	6	BaseRecal_MT
	6	BaseRecal_genome
	6	IndelRealigner_MT
	6	IndelRealigner_genome
	3	KeepOnlyVariants
	3	MergeMutectVarscan
	3	MergeWriteVariants
	3	MutectSplitVariants
	6	PostRecalTable_MT
	6	PostRecalTable_genome
	6	PrintReads_MT
	6	PrintReads_genome
	6	RTC_MT
	6	RTC_genome
	3	Remove_segdups
	3	VarscanSplitVariants
	1	all
	3	annovar_MT_mutect
	3	annovar_MT_varscan
	3	annovar_genome_mutect
	3	annovar_genome_varscan
	3	checkMAFs
	3	create_rsID_table
	6	extractUnmapped_convert
	6	extractUnmapped_extract
	4	fastqc_trimmed_R1
	4	fastqc_trimmed_R2
	3	filterExonicPolymorphic
	3	filterSomatic_Indel
	3	filterSomatic_SNV
	6	indexing_MT
	6	indexing_genome
	6	map_to_MT
	5	map_to_exome
	4	map_to_genome
	6	marking_MT
	6	marking_genome
	3	muTect_MT
	3	muTect_genome
	6	sorting_MT
	6	sorting_exome
	6	sorting_genome
	2	trim
	3	varscan_MT_patient
	6	varscan_MT_sample
	3	varscan_genome_patient
	6	varscan_genome_sample
	221

Job 11: >> B2044S : fastqc2 on trimmed fastq


Job 282: >> B2044D : Aligning to reference EXOME


Job 184: >> 1905 : sorting genome


Job 8: >> B2044S : fastqc1 on trimmed fastq

Activating conda environment /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969.
Activating conda environment /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969.
Activating conda environment /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969.
Activating conda environment /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969.

Job 49: >> 3165 : Trimming


Job 43: >> 273 : Trimming


Job 281: >> 1905 : Aligning to reference EXOME

Activating conda environment /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969.

Job 280: >> B2044S : Aligning to reference EXOME

Activating conda environment /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969.

Job 17: >> B2044D : fastqc1 on trimmed fastq

Activating conda environment /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969.

Job 219: >> B2044D : Aligning to reference NUCLEAR GENOME

Activating conda environment /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969.

Job 181: >> 218 : sorting genome

Activating conda environment /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969.

Job 212: >> B2044S : Aligning to reference NUCLEAR GENOME

Activating conda environment /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969.

Job 277: >> 218 : sorting exome

Activating conda environment /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969.

Job 2: >> B2044D : fastqc2 on trimmed fastq

Activating conda environment /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969.
Waiting at most 5 seconds for missing files.
Will exit after finishing currently running jobs.
Finished job 2.
1 of 221 steps (0.45%) done
Waiting at most 5 seconds for missing files.
Will exit after finishing currently running jobs.
Waiting at most 5 seconds for missing files.
Will exit after finishing currently running jobs.
Error in rule map_to_exome:
    jobid: 282
    output: /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/04_alignment_exome/01_intermediate/B2044D.sam
    log: /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/04_alignment_exome/logs/B2044D_alignment.log

RuleException:
CalledProcessError in line 831 of /home/PERSONALE/eugenio.fonzi2/WESPipeline/snakefile_04.py:
Command 'source activate /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969; set -euo pipefail;  bwa mem -M -t 4 -R '@RG\tID:B2044D\tSM:B2044D\tPL:illumina\tLB:nextera_expanded\tPU:PE' /home/PERSONALE/eugenio.fonzi2/data_ref/nextera_expanded.fasta /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/B2044D_R1.fastq /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/B2044D_R2.fastq > /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/04_alignment_exome/01_intermediate/B2044D.sam 2> /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/04_alignment_exome/logs/B2044D_alignment.log ' returned non-zero exit status 1.
  File "/home/PERSONALE/eugenio.fonzi2/WESPipeline/snakefile_04.py", line 831, in __rule_map_to_exome
  File "/home/PERSONALE/eugenio.fonzi2/miniconda3/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Removing output files of failed job map_to_exome since they might be corrupted:
/home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/04_alignment_exome/01_intermediate/B2044D.sam
Error in rule map_to_exome:
    jobid: 280
    output: /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/04_alignment_exome/01_intermediate/B2044S.sam
    log: /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/04_alignment_exome/logs/B2044S_alignment.log

RuleException:
CalledProcessError in line 831 of /home/PERSONALE/eugenio.fonzi2/WESPipeline/snakefile_04.py:
Command 'source activate /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969; set -euo pipefail;  bwa mem -M -t 4 -R '@RG\tID:B2044S\tSM:B2044S\tPL:illumina\tLB:nextera_expanded\tPU:PE' /home/PERSONALE/eugenio.fonzi2/data_ref/nextera_expanded.fasta /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/B2044S_R1.fastq /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/B2044S_R2.fastq > /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/04_alignment_exome/01_intermediate/B2044S.sam 2> /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/04_alignment_exome/logs/B2044S_alignment.log ' returned non-zero exit status 1.
  File "/home/PERSONALE/eugenio.fonzi2/WESPipeline/snakefile_04.py", line 831, in __rule_map_to_exome
  File "/home/PERSONALE/eugenio.fonzi2/miniconda3/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Removing output files of failed job map_to_exome since they might be corrupted:
/home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/04_alignment_exome/01_intermediate/B2044S.sam
Error in rule map_to_genome:
    jobid: 219
    output: /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/03_alignment_genome/01_intermediate/B2044D.sam
    log: /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/03_alignment_genome/logs/B2044D_alignment.log

RuleException:
CalledProcessError in line 566 of /home/PERSONALE/eugenio.fonzi2/WESPipeline/snakefile_04.py:
Command 'source activate /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969; set -euo pipefail;  bwa mem -M -t 4 -R '@RG\tID:B2044D\tSM:B2044D\tPL:illumina\tLB:nextera_expanded\tPU:PE' /home/PERSONALE/eugenio.fonzi2/data_ref/human_g1k_v37.fasta /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/B2044D_R1.fastq /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/B2044D_R2.fastq > /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/03_alignment_genome/01_intermediate/B2044D.sam 2> /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/03_alignment_genome/logs/B2044D_alignment.log ' returned non-zero exit status 1.
  File "/home/PERSONALE/eugenio.fonzi2/WESPipeline/snakefile_04.py", line 566, in __rule_map_to_genome
  File "/home/PERSONALE/eugenio.fonzi2/miniconda3/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Error in rule map_to_genome:
    jobid: 212
    output: /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/03_alignment_genome/01_intermediate/B2044S.sam
    log: /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/03_alignment_genome/logs/B2044S_alignment.log

RuleException:
CalledProcessError in line 566 of /home/PERSONALE/eugenio.fonzi2/WESPipeline/snakefile_04.py:
Command 'source activate /home/PERSONALE/eugenio.fonzi2/WESPipeline/.snakemake/conda/e35b1969; set -euo pipefail;  bwa mem -M -t 4 -R '@RG\tID:B2044S\tSM:B2044S\tPL:illumina\tLB:nextera_expanded\tPU:PE' /home/PERSONALE/eugenio.fonzi2/data_ref/human_g1k_v37.fasta /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/B2044S_R1.fastq /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/B2044S_R2.fastq > /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/03_alignment_genome/01_intermediate/B2044S.sam 2> /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/03_alignment_genome/logs/B2044S_alignment.log ' returned non-zero exit status 1.
  File "/home/PERSONALE/eugenio.fonzi2/WESPipeline/snakefile_04.py", line 566, in __rule_map_to_genome
  File "/home/PERSONALE/eugenio.fonzi2/miniconda3/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Removing temporary output file /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/04_alignment_exome/01_intermediate/218.sam.
Finished job 277.
2 of 221 steps (0.90%) done
Will exit after finishing currently running jobs.
Removing temporary output file /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/03_alignment_genome/01_intermediate/1905.sam.
Finished job 184.
3 of 221 steps (1%) done
Removing temporary output file /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/03_alignment_genome/01_intermediate/218.sam.
Finished job 181.
4 of 221 steps (2%) done
Removing output files of failed job map_to_genome since they might be corrupted:
/home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/03_alignment_genome/01_intermediate/B2044D.sam
Removing output files of failed job map_to_genome since they might be corrupted:
/home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/03_alignment_genome/01_intermediate/B2044S.sam
Will exit after finishing currently running jobs.
Removing temporary output file /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/1905_R2.fastq.
Removing temporary output file /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/1905_R1.fastq.
Finished job 281.
5 of 221 steps (2%) done
Will exit after finishing currently running jobs.
Will exit after finishing currently running jobs.
Will exit after finishing currently running jobs.
Will exit after finishing currently running jobs.
Will exit after finishing currently running jobs.
Will exit after finishing currently running jobs.
Removing temporary output file /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/01_fastq/02_postprocess/273_R1.fastq.
Removing temporary output file /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/01_fastq/02_postprocess/273_R2.fastq.
Finished job 43.
6 of 221 steps (3%) done
Will exit after finishing currently running jobs.
Terminating processes on user request.
Error in rule trim:
    jobid: 49
    output: /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/3165_R1.fastq, /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/3165_R2.fastq

RuleException:
CalledProcessError in line 497 of /home/PERSONALE/eugenio.fonzi2/WESPipeline/snakefile_04.py:
Command ' set -euo pipefail;  /home/PERSONALE/eugenio.fonzi2/softwares/adapter_removal_1.5.4/AdapterRemoval --file1 /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/01_fastq/02_postprocess/3165_R1.fastq --file2 /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/01_fastq/02_postprocess/3165_R2.fastq --stats --trimns --trimqualities --minquality 20 --minlength 79 --output1 /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/3165_R1.fastq --output2 /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/3165_R2.fastq ' died with <Signals.SIGINT: 2>.
  File "/home/PERSONALE/eugenio.fonzi2/WESPipeline/snakefile_04.py", line 497, in __rule_trim
  File "/home/PERSONALE/eugenio.fonzi2/miniconda3/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Removing output files of failed job trim since they might be corrupted:
/home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/3165_R1.fastq, /home/PERSONALE/eugenio.fonzi2/ANALYSES_WES/180105_test_adapters_NO_on_contaminated/02_fastq_trimmed/3165_R2.fastq
Complete log: .snakemake/log/2018-01-07T061303.057908.snakemake.log
